src/System.Globalization/tests/CultureInfo/CultureInfoConstructor2.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using Xunit;
+
+namespace System.Globalization.Tests
+{
+    public class CultureInfoConstructor2
+    {
+        [Fact]
+        public void TestDiffCulture()
+        {
+            CultureInfo myCulture = new CultureInfo("");
+            CultureInfo myCultureEn = new CultureInfo("en");
+            CultureInfo myCultureFr = new CultureInfo("fr-FR");
+            CultureInfo myCultureDe = new CultureInfo("de-DE");
```

Comment:
What is this testing, just that the ctor doesn't throw?  Should it be validating any properties on the resulting CultureInfo?